src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.header {
+  @include flexSpaceCenter;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  background-color: $color-white;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
```

Comment:
It looks like we are missing `nav` block itself, but there are elements that belong to that block 🤔

Reply:
Not resolved